app/survey_config/business_config.py

```diff
@@ -134,10 +132,9 @@ def _stripped_base_url(self) -> str:
 
 @dataclass
 class NorthernIrelandBusinessSurveyConfig(BusinessSurveyConfig):
-
-    page_header_logo: str = "ni-finance-logo"
-    page_header_logo_alt: str = lazy_gettext(
-        "Northern Ireland Department of Finance logo"
-    )
-    mobile_logo: str = "ni-finance-logo-mobile"
-    custom_header_logo: bool = True
+    masthead_logo: Optional[
```

Comment:
Can these svg assets be separated from the module? I don't think we are allowed to store such long vars in the code.